plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduClientSession.java

```diff
@@ -619,4 +621,11 @@ private KuduSplit toKuduSplit(KuduTableHandle tableHandle, KuduScanToken token,
             throw new TrinoException(GENERIC_INTERNAL_ERROR, e);
         }
     }
+
+    @PreDestroy
+    public void close()
+        throws KuduException
```

Comment:
Checkstyle is failing due to incorrect indentation.

Reply:
Apologies - I saw the failure and amended the commit, but forgot to push :-/
Thanks for spotting it!